src/receipts.rs

```diff
@@ -76,8 +76,8 @@ pub struct ReceiptSigner {
 impl ReceiptSigner {
     pub fn new(signer: PrivateKeySigner, chain_id: U256, verifying_contract: Address) -> Self {
         let v2_domain = Eip712Domain {
-            name: Some("TAP".into()),
-            version: Some("2".into()),
+            name: Some("GraphTallyCollector".into()),
```

Comment:
Should it be `GraphTally`?  without "Collector" to align with [tap-core](https://github.com/semiotic-ai/timeline-aggregation-protocol/pull/325/files)